org.eclipse.transformer.cli/src/test/java/transformer/test/TestCommandLine.java

```diff
@@ -340,6 +341,60 @@ void testSetLogFile() throws Exception {
 
 	}
 
+	@Test
+	void testEmptyInputRejected() throws Exception {
+		String inputFileName = "";
+		String outputFileName = DYNAMIC_CONTENT_DIR + '/' + "foo";
+		verifyActionInvalidEmptyInputRejected(DirectoryActionImpl.class.getName(), inputFileName, outputFileName);
+	}
+
+	@Test
+	void testEmptyOutputRejected() throws Exception {
+		String inputFileName = STATIC_CONTENT_DIR + "/command-line/A.java";
+		String outputFileName = "";
+		verifyActionInvalidEmptyInputRejected(DirectoryActionImpl.class.getName(), inputFileName, outputFileName);
+	}
```

Comment:
Use parameterize test instead of pulling common asserts out into a private method. The only thing that varies are the parameters.

Reply:
Sorry for the delayed response. 
Parameterization of this is challenging because DYNAMIC_CONTENT_DIR is dynamically modified each test. 
Do you have any ideas?